lib/matplotlib/widgets.py

```diff
@@ -1679,61 +1679,65 @@ def set_visible(self, visible):
 
 class SpanSelector(_SelectorWidget):
     """
-    Select a min/max range of the x or y axes for a matplotlib Axes.
+    Visually select a min/max range on a single axis and call a function with
+    those values.
 
-    For the selector to remain responsive you must keep a reference to
+    To guarantee that the selector remains responsive, keep a reference to
     it.
 
-    Example usage::
+    In order to turn off the SpanSelector, set `span_selector.active=False`. To
+    turn it back on, set `span_selector.active=True`.
 
-        ax = subplot(111)
-        ax.plot(x,y)
-
-        def onselect(vmin, vmax):
-            print(vmin, vmax)
-        span = SpanSelector(ax, onselect, 'horizontal')
-
-    *onmove_callback* is an optional callback that is called on mouse
-    move within the span range
+    Parameters
+    ----------
+    ax :  :class:`matplotlib.axes.Axes` object
 
-    """
+    onselect : function of form func(min, max), where min/max are floats
 
-    def __init__(self, ax, onselect, direction, minspan=None, useblit=False,
-                 rectprops=None, onmove_callback=None, span_stays=False,
-                 button=None):
-        """
-        Create a span selector in *ax*.  When a selection is made, clear
-        the span and call *onselect* with::
+    direction : "horizontal" or "vertical"
+      The axis along which to draw the span selector
 
-            onselect(vmin, vmax)
+    minspan : float, default is None
+     If selection is less than *minspan*, do not call *onselect*
 
-        and clear the span.
+    useblit : bool, default is False
+      If True, use the backend-dependent blitting features for faster
+      canvas updates. Only available for GTKAgg right now.
 
-        *direction* must be 'horizontal' or 'vertical'
+    rectprops : dict, default is None
+      Dictionary of :class:`matplotlib.patches.Patch` properties
 
-        If *minspan* is not *None*, ignore events smaller than *minspan*
+    onmove_callback : function of form func(min, max), where min/max are floats, default is None
```

Comment:
I am pretty sure this line is too long for the pep8 requirements (to be confirmed when travis finish running).